articles/connections/database/custom-db/templates/change-password.md

```diff
@@ -486,9 +473,9 @@ function changePassword (email, newPassword, callback) {
   connection.on('connect', function (err) {
     if (err) return callback(err);
 
-    var request = new Request(query, function (err, rows) {
+    const request = new Request(query, function (err, rows) {
       if (err) return callback(err);
-      console.log('rows: ' + rows);
+      // console.log('rows: ' + rows);
```

Comment:
This line probably should be removed from both the db-templates and the doc example.

Reply:
Done